data_anonymization_s3/README.md

```diff
@@ -0,0 +1,48 @@
+# Data Anonymization
+
+Example of a mse applications to anonymize data.
```

Comment:
The specificity of this example is S3 right? So what about talking about it a bit more?